README.md

```diff
@@ -1 +1,76 @@
-# awesome-books-v1
\ No newline at end of file
+# Awesome-books
+
+This project is focused on building  is a simple website that displays a list of books that users have added.It allows you to add and remove books from that list also. The goal of this project is to build a simple multipage page responsive book website.
+
+In this project, you are expected to setup Html-CSS-Javascript linter following the guildlines stated in the Html-CSS-Javascript limter config documentation, follow mobile first approach while building the website, make the website responsive on mobile and desktop, make website interactive, run tests and finally deploy.
+
+The project will contain the List, Add and Contact page.
+
+## Built With
+
+- HTML5,
+- Javascript
+- CSS3
+## Live Demo
+
+<!-- [Live Demo Link](https://kendoriddy.github.io/awesome-books/) -->
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+
+## Prerequisites
+- Have a good knowledge on how to parse UI designs
+- Have VSCode or other text editor installed. [Link to download VSCode](https://code.visualstudio.com/download)
+- Install node package. [Link to download node](https://nodejs.org/en/download/)
+- Have git installed.[Link to download git](https://git-scm.com/downloads)
+- Create a github repository.
+
+## Setup
+- git clone https://github.com/kendoriddy/awesome-books-v1.git
+- cd awesome-books-v1/
+-Follow the HTML-CSS linter configuration to docs to setup linters [Link to Microverse linter docs](https://github.com/microverseinc/linters-config/tree/master/html-css-js)
+
+## Run tests
+
+For tracking linters errors locally, you need to follow these steps:
+
+- For tracking linter errors in HTML file run:
+`npx hint .`
+- For tracking linter errors in CSS file run:
+`npx stylelint "**/*.{css,scss}"`
+- For tracking linter errors in Javascript file run:
+`npx eslint .`
+
+## Deployment
+
+- Use GitHub pages to deploy website
+- For more information about publishing sources, see [About github pages](https://pages.github.com/)
+
+## Contributors
+
+## Author 1
+
+👤 **Kehinde Ridwan Onifade**
+
+- GitHub: [Kenny](https://github.com/kendoriddy)
+- LinkedIn: [Kenny](https://www.linkedin.com/in/kehinde-onifade)
+
+## Autor 2
+**Ernest Lasten**
+- GitHub: [@githubhandle]
+- Twitter: [@twitterhandle]
```

Comment:
- [ ] Please add the GitHub and Twitter handle of the other partner. :+1: There are missing in your readme but this partner is contributing equally in this project right :thinking: